eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -150,12 +150,12 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
       forwarder ! remote
       val open = OpenChannel(nodeParams.chainHash,
         temporaryChannelId = temporaryChannelId,
-        fundingSatoshis = fundingSatoshis,
-        pushMsat = pushMsat,
-        dustLimitSatoshis = localParams.dustLimitSatoshis,
+        fundingSatoshis = Satoshi(fundingSatoshis),
```

Comment:
It's going to be a bit painful, but as we correctly type these amounts maybe we should rename the variables, they don't need to explicitly name the unit anymore?
`fundingSatoshis -> fundingAmount`
`pushMsat -> pushAmount`
`dustLimitSatoshis -> dustLimit`
etc?

Reply:
I thought about that but we have to consider those objects map messages from the spec and there is value in keeping exactly the same name. Will definitely rename variables that are not part of the spec messages.

Reply:
INPUT_INIT_FUNDER sat/msat fields should be typed, same will all command messages